crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -441,38 +441,42 @@ pub mod state_update {
     }
 
     /// L2 contract data within state diff.
-    #[derive(Clone, Debug, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+    #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
     #[serde(deny_unknown_fields)]
-    pub struct Contract {
+    pub struct DeployedContract {
         pub address: ContractAddress,
         /// `class_hash` is the field name from cairo 0.9.0 onwards
-        /// `contract_hash` is the name for cairo before 0.9.0
-        #[serde(alias = "class_hash")]
-        pub contract_hash: ClassHash,
+        /// `contract_hash` is the name from cairo before 0.9.0
+        #[serde(alias = "contract_hash")]
+        pub class_hash: ClassHash,
```

Comment:
I've added this change to decrease confusion and stick to the current convention, which is:
- always indicating the difference between declared and deployed,
- class_hash is the correct term, contract_hash is the deprecated one.